.config/forest.dic

```diff
@@ -17,14 +18,18 @@ cron
 crypto
 daemon
 daemonize
+DAG_CBOR
+DefaultParams
 Filecoin
+fvm_ipld_encoding
```

Comment:
This is code and not English words. They belong in code blocks rather than the dictionary.

Reply:
fixed